get-involved.php

```diff
@@ -109,9 +109,9 @@
         </div>
         <div class="half">
             <h3>Question &amp; Answer</h3>
-            <p>Stack Exchange is a Q&amp;A website built around finding the best answers to common questions. Anyone can create an account to start asking and answering.</p>
-            <a class="button flat" href="https://elementaryos.stackexchange.com/questions?sort=newest">New Questions</a>
-            <a class="button flat suggested-action" href="https://elementaryos.stackexchange.com/unanswered">Unanswered Questions</a>
+            <p>We use GitHub Discussions for community Q&A. Anyone can create an account to start asking and answering.</p>
+            <a class="button flat" href="https://github.com/orgs/elementary/discussions/categories/q-a?discussions_q=category:Q&A+sort:date_created">New Questions</a>
```

Comment:
I'm testing both of these links and it looks like they are not filtering something correctly when they open discussions. For each of these, I see a screen that says there are no discussions. I know that there ARE discussions for elementary so I'm not sure if thats what we want.

Reply:
Ah I figured it out. The ampersand ha